one-pager-maker/src/stories/Usertem.tsx

```diff
@@ -0,0 +1,28 @@
+import {useState} from "react";
+import {Menu, MenuItem} from "./Menu.tsx";
+import {useDetectClickOutside} from "react-detect-click-outside";
+
+interface UserItemProps {
+    userName: string;
+    onClick: (user: string) => void;
+}
+export const UserItem: React.FC<UserItemProps> = ({userName, onClick}) => {
```

Comment:
You may not need to use `React.FC<T>`. Try to use `JSX.Element` instead.
```TypeScript
export const UserItem = ({userName, onClick}: UserItemProps) => {

}
```
reference: https://zenn.dev/motonosuke/articles/13d0fdd417ec7c

Reply:
fixed